config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "server_addr": "0.0.0.0:3300",
+    "tmp_file_dir": "/tmp/agar_service/",
+    "latexmk_path": "latexmk"
+}
```

Comment:
Can we override those by setting env variables?

Reply:
Not currently, I can make it so that's a thing.